x-pack/plugins/apm/public/hooks/useTransactionDistribution.ts

```diff
@@ -69,10 +69,12 @@ export function useTransactionDistribution(urlParams: IUrlParams) {
           // selected sample was not found. select a new one:
           // sorted by total number of requests, but only pick
           // from buckets that have samples
+          const bucketsSortedByPreference = response.buckets
```

Comment:
From the name it sounds like the sorting depends on something. But they are always sorted by count, no?

Reply:
Yes - will change it to bucketsSortedByCount.